testing/test_sliceFilter.py

```diff
@@ -76,4 +76,4 @@ def test_slice_filter(transforms_dict, train_lst, target_lst, roi_params, slice_
         assert cmpt_neg == 0
         assert cmpt_pos != 0
     else:
-        assert cmpt_neg > cmpt_pos
+        assert cmpt_neg != 0 and cmpt_pos != 0
```

Comment:
Images changes so the balance as well.

Reply:
Do you mean: the new testing image as more pos than neg? ok
I feel this "assert" is a bit useless.. what about: ` cmpt_neg < cmpt_pos` --> ie no filtering has been done.

Reply:
Since the test is designed to filter negative slice, I am just checking if there a still some negative examples.